files/Trimble_Tekla/Sprint_2_1.ifc

```diff
@@ -0,0 +1,113 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition[Undefined]'),'2;1');
+FILE_NAME('Sprint_2_1.ifc','2023-04-14T12:54:03',('Author/creator of the IFC file'),('Trimble Inc.'),'IfcExportSDK2 ver. 4.7.1','Trimble Software - Version xx.x','Authorizer of the IFC file');
+FILE_SCHEMA(('IFC4X4'));
+ENDSEC;
+
+DATA;
+#1= IFCPOSTALADDRESS(.HOME.,'My postal address',$,$,('StreetAddress 1','StreetAddress 2'),'Box 1','Town','Region','Box 1','Country');
+#2= IFCACTORROLE(.CONSULTANT.,'Role','Role Description');
+#3= IFCPERSON('HLahtel','Family name','Given Name',$,$,$,(#2),(#1));
+#4= IFCACTORROLE(.CONSULTANT.,$,$);
+#5= IFCPOSTALADDRESS(.OFFICE.,'Office postal address',$,$,('StreetAddress 1','StreetAddress 2'),'Box 1','Town','Region','Box 1','Country');
+#6= IFCORGANIZATION('TRMB','Trimble Inc.','Trimble consulting',(#4),(#5));
+#7= IFCPERSONANDORGANIZATION(#3,#6,$);
+#8= IFCAPPLICATION(#6,'version','Application full name','Application Identifier');
+#9= IFCOWNERHISTORY(#7,#8,$,.NOCHANGE.,$,$,$,1681466043);
+#10= IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#11= IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#12= IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#13= IFCSIUNIT(*,.MASSUNIT.,.KILO.,.GRAM.);
+#14= IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
+#15= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#16= IFCSIUNIT(*,.SOLIDANGLEUNIT.,$,.STERADIAN.);
+#17= IFCSIUNIT(*,.POWERUNIT.,$,.WATT.);
+#18= IFCUNITASSIGNMENT((#10,#11,#12,#13,#14,#15,#16,#17));
+#19= IFCCARTESIANPOINT((-200.,200.,0.));
+#20= IFCDIRECTION((0.,0.,1.));
+#21= IFCDIRECTION((1.,0.,0.));
+#22= IFCAXIS2PLACEMENT3D(#19,#20,#21);
+#23= IFCDIRECTION((0.725975,0.687721));
+#24= IFCGEOMETRICREPRESENTATIONCONTEXT('3D','Model',3,1.E-06,#22,#23);
+#25= IFCPROPERTYSINGLEVALUE('Project data',$,IFCLABEL('Project data 1.'),$);
+#26= IFCPROPERTYSINGLEVALUE('Length property',$,IFCLENGTHMEASURE(1.),#10);
+#27= IFCPROPERTYSET('2VvPInIaH4DwJhCV6wOt4Z',#9,'Project pset',$,(#25,#26));
+#28= IFCARCHELEMENTTYPE('15kDej$Jj5WRaa0jByCU3H',#9,'Name',$,$,$,$,$,$,.NOTDEFINED.);
+#29= IFCGROUP('1U1QQhW69A_8bBUGHUmG8K',#9,'TypeObject Group',$,$);
+#30= IFCRELASSIGNSTOGROUP('1rf9RG57H9jxvJeMb$RIi_',#9,$,$,(#28),$,#29);

```

Comment:
The reason why the checker complains here, is:

- One can group `IfcObjectDefinition` in a group following the schema.
- One can only group `IfcProduct` following the [concept template](https://standards.buildingsmart.org/IFC/RELEASE/IFC4/ADD2_TC1/HTML/link/group-assignment.htm) - `IfcArchElementType` is not `IfcProduct`.

Reply:
OK. I'll fix it.